nuget/readme.md

```diff
@@ -147,7 +147,7 @@ For example, if the verbosity is set to minimal, then only messages with high im
 The default importance of C++/WinRT build messages is 'normal', but this can be overridden with the CppWinRTVerbosity property to enable throttling of C++/WinRT messages independent of the overall verbosity level.
 
 Example:
-> msbuild project.vcxproj /vebosity:minimal /property:CppWinRTVerbosity=high ...
+> msbuild project.vcxproj /verbosity:minimal /property:CppWinRTVerbosity=high ...
```

Comment:
It's _possible_ this typo is intentional.

AFAICT, MSVC misspelled `Namespace` as shown here:
https://github.com/microsoft/cppwinrt/blob/cbdd01c9953e91f5a983c05aadf05172deec6023/.github/actions/spelling/patterns.txt#L28
(I don't have a handy feedback portal to MS teams, so sometimes I ask unrelated MS folk to lean on people. It's ok if it isn't fixed, I've added it to [spell-check-this](https://github.com/check-spelling/spell-check-this/tree/prerelease/) so people can ignore it....)

Reply:
I think this is a legit typo since it's a comment. 

```
D:\git>msbuild /vebosity
MSBuild version 17.9.5+33de0b227 for .NET Framework
MSBUILD : error MSB1001: Unknown switch.
    Full command line: 'msbuild  /vebosity'
  Switches appended by response files:
'' came from 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\MSBuild.rsp'
Switch: /vebosity

For switch syntax, type "MSBuild -help"

D:\git>msbuild /verbosity
MSBuild version 17.9.5+33de0b227 for .NET Framework
MSBUILD : error MSB1016: Specify the verbosity level.
    Full command line: 'msbuild  /verbosity'
  Switches appended by response files:
'' came from 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\MSBuild.rsp'
Switch: /verbosity

For switch syntax, type "MSBuild -help"
```